relay_test.go

```diff
@@ -1716,7 +1716,8 @@ func TestRelayModifyArg2OnFrameWithFragmentedArg2ShouldFail(t *testing.T) {
 				"fum": testutils.RandString(16 * 1024),
 			}),
 		})
-		assert.EqualError(t, err, "tchannel error ErrCodeTimeout: timeout")
```

Comment:
ah, somehow this didn't stand out when we were testing this....

Reply:
yeah I totally missed this too!